skietbaan_fe/src/components/LoginStyles.css

```diff
@@ -38,8 +38,28 @@ body{
   top:25%
 }
 }
+input.invalid{
+  border:1px solid red;
+}
+.invalid-message{
+  color:red;
+  font-size: 12px;
+}
+button{
+  border-radius: 30px !important;
```

Comment:
Avoid using !important in CSS. Understand first why it is not rendering the desired radius and then see if the parent container of the element is hindering it.
!important should be your last arsenal just a knife is the last tool on a soldier.

Reply:
once everything is important nothing is
![image](https://user-images.githubusercontent.com/47351935/52393173-aa840b80-2aad-11e9-95bc-780c58883243.png)

Reply:
> Avoid using !important in CSS. Understand first why it is not rendering the desired radius and then see if the parent container of the element is hindering it.
> !important should be your last arsenal just like a knife is the last tool on a soldier.

Thank you, you @laurent-mbuyu .This makes me angry as well :)

Reply:
ok, noted